MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -8682,4 +8689,90 @@ public IUnitRating getUnitRating() {
 		
 		return unitRating;
 	}
+
+    public int getPeacetimeCost() {
+        int cost = 0;
+
+        cost += getPayRoll(getCampaignOptions().useInfantryDontCount());
+        cost += getMonthlySpareParts();
+        cost += getMonthlyFuel();
+        cost += getMonthlyAmmo();
+        return cost;
+    }
+
+    public long getMonthlySpareParts() {
```

Comment:
Can be private.

Reply:
That was part of the peacetime costs feature